WhatIsASprint.md

```diff
@@ -0,0 +1,23 @@
+
+# **What is a Sprint?**
+
+
+A **_sprint_** is a set period of time broken out from a project's overall timeline with associated tasks to be completed during that period of time. That list of tasks is referred to as the sprint's <b>Product Backlog</b>.
+
+
+A *sprint* allows a larger timeline to be easily broken into smaller, more digestible pieces in order to focus effort from all stakeholders on a few logically scheduled tasks at a time.
+
+Generally all deliverables from a sprint must be accepted by the key stakeholders prior to moving to the next sprint and a project will have multiple sprints until scope has been met.
+
+A *sprint* generally begins with a planning meeting where the person or persons requesting the work meet with the developers in order to determine what work can realistically be completed during the time period, or sprint, being discussed. The developer will generally have the final say on what work will be completed during the sprint.
+
+In some organizations these meetings will be overseen by either a Scrum Master or a Project Manager in order to ensure the overall goal of the project has been met after all sprints have been completed.
```

Comment:
It is also the first time I am using the term 'Project Manager' also 'Product Backlog'. I would like to come back later once I have written those articles and add  those as links to other FCC wiki's.